src/components/scenes/CreateWalletCompletionScene.tsx

```diff
@@ -160,6 +169,40 @@ const CreateWalletCompletionComponent = (props: Props) => {
 
   const handleNext = useHandler(() => navigation.navigate('walletsTab', { screen: 'walletList' }))
 
+  const handleMigrate = useHandler(() => {
+    // Transform filtered items into the structure expected by the migration component
+    const migrateWalletList: MigrateWalletItem[] = newWalletItems.map(createWallet => {
+      // Link the createWalletIds with the created wallets
+      const { currencyCode, key, pluginId, tokenId, walletType } = createWallet
+      const wallet = wallets.find(wallet => wallet.currencyInfo.pluginId === pluginId)
+
+      return {
+        createWalletIds: wallet == null ? [''] : [wallet.id],
+        currencyCode,
+        displayName: walletNames[key],
+        key,
+        pluginId: pluginId,
+        tokenId: tokenId,
+        walletType: walletType,
+        type: 'create'
+      }
+    })
+
+    // Navigate to the migration screen with the prepared list
+    if (migrateWalletList.length > 0) {
+      // TODO: Waiting for wallet sync still yields 0 getMaxSpendable...
+      // navigation.navigate('migrateWalletCalculateFee', {
+      //   migrateWalletList
+      // })
+
+      navigation.navigate('migrateWalletSelectCrypto', {
```

Comment:
It seems like the migrateWalletSelectCrypto scene is being used to delay the user so hopefully the wallets finish syncing by the time they get to the migrateWalletCalculateFee scene. Maybe be better to instead wait for sync ratio and then pause after it briefly to ensure the balance is ready. That seems more likely to be successful than adding the extra scene to slow them down. 

If the routing to the select scene stays or not is ultimately not that important but I dod think where a user waits for the wallet to sync is. That should be in the calculate fee scene before getMaxSpendable is called since there's already a spinner.